conn.go

```diff
@@ -259,8 +261,8 @@ type Conn struct {
 	readErrCount  int
 	messageReader *messageReader // the current low-level reader
 
-	readDecompress         bool // whether last read frame had RSV1 set
-	newDecompressionReader func(io.Reader) io.ReadCloser
+	readDecompress         bool                          // whether last read frame had RSV1 set
+	newDecompressionReader func(io.Reader) io.ReadCloser // arges may flateReadWrapper struct
```

Comment:
I don't understand the code comment here. What "arges may flateReadWrapper struct" mean ?

Reply:
sorry, this comment is unnecessary.
This was left for myself during initial development.

thanks.